app/js/components/queryBuilder/queryBuilder.component.js

```diff
@@ -200,20 +199,21 @@ const QueryBuilderComponent = {
                     vm[`${sortParams.set}Set`] = (_setIsSorted(list, sortedList, sortOn) ? list.reverse() : sortedList);
                     return (_setIsSorted(list, sortedList, sortOn) ? list.reverse() : sortedList);
                   };
-            
-                  
+
+
 
 
                 /**
-                 * @param  {Object} queryParam - mutation or DiseaseModel   
-                 * @return {Array | false} Array of objects if queryParam is added, 
+                 * @param  {Object} queryParam - Array of mutations or DiseaseModels
```

Comment:
I found this comment line misleading:
(1) The parameter should be `queryParamData`;
(2) The type of `queryParamData` should be an `Array` instead of `Object` (although technically speaking `Array` is also an `Object`.)